### ISSUE TITLE
Yesterday I use this for the first time, and I got VAC banned.

### ISSUE THREAD

**Issue by faroperfect** [danielkrupinski/Osiris#4030]
This project can not bypass VAC any more obviously.

**Comment by prolikeme** [comment]
Nah. Either you're doing it wrong or your injector is detected. Been using it for almost 1 month until now and no vac and im still in green trust factor. *Sry for my bad english*

**Comment by Xsintashi** [comment]
congratz. you got banned like everyone who used it at least once. This project is not undetected cuz its public free cheat on github.
also you didnt use vac bypass

**Comment by prolikeme** [comment]
> congratz. you got banned like everyone who used it at least once. This project is not undetected cuz its public free cheat on github. also you didnt use vac bypass

Mine still green trust factor and no vac. using it for almost a month now. you just gotta use vac bypass and a good injector. if i get red trust or vac im gonna post or update here

**Comment by Adim2P** [comment]
> > congratz. you got banned like everyone who used it at least once. This project is not undetected cuz its public free cheat on github. also you didnt use vac bypass
> 
> Mine still green trust factor and no vac. using it for almost a month now. you just gotta use vac bypass and a good injector. if i get red trust or vac im gonna post or update here

Do you use Daniel's VAC Bypass or something else?

**Comment by marian-dev0** [comment]
I uploaded the updated version here:
[https://github.com/marian-dev0/CSGO-Mod-Menu](https://github.com/marian-dev0/CSGO-Mod-Menu)

**Comment by zhnkc9** [comment]
> > congratz. you got banned like everyone who used it at least once. This project is not undetected cuz its public free cheat on github. also you didnt use vac bypass
> 
> Mine still green trust factor and no vac. using it for almost a month now. you just gotta use vac bypass and a good injector. if i get red trust or vac im gonna post or update here

bro, could you please send me your injector, your parameters, and your DLL for reference? The known issue I have found so far is that 'back trace' will definitely be banned (my ping is 5ms, HKServer, and my backtracking is 200ms). Can we discuss it together?

**Comment by zhnkc9** [comment]
> > 恭喜。您像每个至少使用过一次的人一样被禁止。这个项目并非未被发现，因为它在 github 上公开免费作弊。你也没有使用真空旁路
> 
> 我的仍然是绿色信任因素，没有真空。现在使用它快一个月了。你只需要使用真空旁路和一个好的注射器。如果我得到 red trust 或 vac 我会在这里发布或更新

兄台，我这边开了就红信，昨天能确定是回溯肯定有问题，可以一起交流一下么。dll我用的是最新的代码编译的。

**Comment by zhnkc9** [comment]
> > 恭喜。你像每个至少使用过一次的人一样被禁止。这个项目并没有被发现，因为它在github上公开免费作弊。你也没有有使用真空旁路
> 
> 我的仍然是绿色信任原因，没有真空。现在使用它快一个月了。你只需要使用真空旁路和一个好的注射器。如果我得到红色信任或真空我会在这里发布或更新

兄弟，我这边开了就红了，昨天能确定是回源肯定有问题，可以一起交流一下。dll我用的是最新的代码编译的。

**Comment by prolikeme** [comment]
> > > congratz. you got banned like everyone who used it at least once. This project is not undetected cuz its public free cheat on github. also you didnt use vac bypass
> > 
> > 
> > Mine still green trust factor and no vac. using it for almost a month now. you just gotta use vac bypass and a good injector. if i get red trust or vac im gonna post or update here
> 
> bro, could you please send me your injector, your parameters, and your DLL for reference? The known issue I have found so far is that 'back trace' will definitely be banned (my ping is 5ms, HKServer, and my backtracking is 200ms). Can we discuss it together?

Dm me on discord 

weeekly#1670

**Comment by swer45** [comment]
Stay updated, use VMProtector and an updated injector. If you use VAC Bypass you will get instant yellow trust factor.

**Comment by prolikeme** [comment]
> Stay updated, use VMProtect and an updated injector. If you use VAC Bypass you will get instant yellow trust factor.

I always use vac bypass and im still at green trust. use better injector

**Comment by swer45** [comment]
> > Stay updated, use VMProtect and an updated injector. If you use VAC Bypass you will get instant yellow trust factor.
> 
> I always use vac bypass and im still at green trust. use better injector

Which injector do you use?

**Comment by prolikeme** [comment]
> > > Stay updated, use VMProtect and an updated injector. If you use VAC Bypass you will get instant yellow trust factor.
> > 
> > 
> > I always use vac bypass and im still at green trust. use better injector
> 
> Which injector do you use?

Got mine from mpgh and its called "CSGOInjector" with vac bypass

**Comment by danielcool25** [comment]
> > > > Stay updated, use VMProtect and an updated injector. If you use VAC Bypass you will get instant yellow trust factor.
> > > 
> > > 
> > > I always use vac bypass and im still at green trust. use better injector
> > 
> > 
> > Which injector do you use?
> 
> Got mine from mpgh and its called "CSGOInjector" with vac bypass

Can't find it myself. Can you give a link or maybe the author of the injector.

**Comment by gur22-09** [comment]
> This project can not bypass VAC any more obviously.

what did you expect, this is a public free cheat.
since source 2 valve is aggresively tracking hardware ids of cheaters and issuing bans.

any other steam account with the same number will also be banned.

`NEVER EVER CHEAT ON MAIN`

**Comment by prolikeme** [comment]
> > > > > Stay updated, use VMProtect and an updated injector. If you use VAC Bypass you will get instant yellow trust factor.
> > > > 
> > > > 
> > > > I always use vac bypass and im still at green trust. use better injector
> > > 
> > > 
> > > Which injector do you use?
> > 
> > 
> > Got mine from mpgh and its called "CSGOInjector" with vac bypass
> 
> Can't find it myself. Can you give a link or maybe the author of the injector. And does it work fine on prime accounts?

You have discord? pm me and i'll send you and yes ofc it works on prime

weeekly#1670

**Comment by prolikeme** [comment]
> > This project can not bypass VAC any more obviously.
> 
> what did you expect, this is a public free cheat. since source 2 valve is aggresively tracking hardware ids of cheaters and issuing bans.
> 
> any other steam account with the same number will also be banned.
> 
> `NEVER EVER CHEAT ON MAIN`

Idk how you guys are getting banned tho. im using this for more than a month now and im still on green trust. im on my main too lol. just legit cheating no rage

**Comment by Xsintashi** [comment]
playing on main with cheats is the worst what you can do. No matter is it gamesense or osiris. You will reget it sooner or later.

**Comment by prolikeme** [comment]
> playing on main with cheats is the worst what you can do. No matter is it gamesense or osiris. You will reget it sooner or later.

Nah. just playing cs for fun. its not like im aiming to be pro or what lol and im already expecting for my main to be banned sooner or later so no worries
